.github/workflows/release-porting-guide.yml

```
@@ -0,0 +1,54 @@
+---
+name: Ansible porting guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: Release version. For example, 11.1.0
+        required: true
+      ansible-major-version:
+        description: Major version. For example, 11
+        required: true
+
+jobs:
+  upload-porting-guide:
+    name: Extract the porting guide${{ '' }}  # Nest jobs under the same sidebar category
```

Why is that `${{ '' }}` needed, and what does the comment wants to say? Does the comment refer to the templating, or to something else?

---

@felixfontein it's unnecessary in this specific instance. But in general, it's a hack I invented to fight how poorly GHA is rendering matrixes of reusable workflows in the sidebar by default (with multiple collapsed sections for each of the jobs within a matrix).

```suggestion
    name: Extract the porting guide
```